templates/fbbot/message-item.php

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
@igroomgrim separate class file pls~ 
Also, if this one is a class, might be better to consider move it into `includes/fbbot` ?